jobserv/project-schema.yml

```diff
@@ -126,6 +126,12 @@ mapping:
                     mapping:
                       regex;(^[a-zA-Z0-9_\.\-]+$):
                         type: str
+                  shared-volumes:
```

Comment:
<nerd_mode>
what about only one type of volumes - just persistent-volumes, but with additional 'mode' attribute: every volume has name, mount-point and mode(RW by default) and RO mode can be used shared volumes.

This approach requires more code-changes, but it is just an idea.
</nerd_mode>

Reply:
persistent read-write volumes work a little differently. They don't have to configured by the host up front and are instead nested under the project in a dynamic self-contained way.